downstream/modules/platform/proc-add-eda-safe-plugin-var.adoc

```diff
@@ -0,0 +1,25 @@
+
+[id="proc-add-eda-safe-plugin-var"]
+
+= Adding a safe plugin variable to {EDAcontroller}
+
+When using redhat.insights_eda or similar plug-ins to run rulebook activations in {EDAcontroller}, you must add a safe plugin variable to a directory in {PlatformNameShort}. This would ensure connecton between {EDAcontroller} and the source plugin, and display port mappings correctly. 
+
+.Procedure
+
+. Create a directory for the safe plugin variable: `mkdir -p ./group_vars/automationedacontroller`
+. Create a file within that directory for your new setting (for example, `touch ./group_vars/automationedacontroller/custom.yml`)
+. Add the variable `automationedacontroller_additional_settings` to extend the default `settings.yaml` template for {EDAcontroller} and add the *SAFE_PLUGINS* field with a list of plugins to enable. For example: 
++
+----
+automationedacontroller_additional_settings:
+   SAFE_PLUGINS:
+     - ansible.eda.webhook
+     - ansible.eda.alertmanager
+----
++
+[NOTE]
+====
+You can also extend the `automationedacontroller_additional_settings` variable beyond SAFE_PLUGINS in the django configuration file, /etc/ansible-automation-platform/eda/settings.yaml 
```

Comment:
I believe Django is a proper noun and should be capitalized.

Reply:
Thanks! Will handle in separate PR.